lib/CL/devices/almaif/XrtDevice.cc

```diff
@@ -30,20 +30,30 @@
 
 XrtDevice::XrtDevice(char *xrt_kernel_name) {
 
-  char xclbin_char[120];
-  snprintf(xclbin_char, sizeof(xclbin_char), "%s.xclbin", xrt_kernel_name);
-
-  char kernel_char[120];
-  snprintf(kernel_char, sizeof(kernel_char), "%s:{%s_1}", xrt_kernel_name,
-           xrt_kernel_name);
+  unsigned xclbin_char_length = strlen(xrt_kernel_name) + 8;
```

Comment:
This file seems to use the C style although it's C++? Perhaps convert the style at some point.